packages/schematics/angular/universal/index.ts

```diff
@@ -259,6 +261,7 @@ export default function (options: UniversalOptions): Rule {
         ...options as object,
         stripTsExtension: (s: string) => s.replace(/\.ts$/, ''),
         hasLocalizePackage: !!getPackageJsonDependency(host, '@angular/localize'),
+        hasPlatformServerShims: Number(ngMajorVersion) >= 12,
```

Comment:
This check shouldn't be needed, since Schematics v12 are only compatible with Angular FW v12.

Reply:
Yeah. Basically, I left it here as a placeholder for disucssing whether we want to do it in a minor version (in which case we could do a `<major>.<minor>` check).

But if we want to introduce this in a major, then we can remove this.

Reply:
Since, we missed the train for v11.2.x (and there probably won't be another minor before 12.0.0), I am going to remove this check.

Reply:
SGTM